### ISSUE TITLE
Bug: JS WrongFilePathLinter triggering in string templates

### ISSUE THREAD

**Issue by netux** [iljapostnovs/ui5plugin-linter#136]
**Describe the bug**
The JS WrongFilePathLinter may trip up when trying to find the file of a class path generated dynamically using string literals.

In my case, I have the following function:
```js
loadFragmentRelative(fragmentName) {
  return this.loadFragment({ name: `com.mycompany.myapp.view.${fragmentName}` });
}
```

**To Reproduce**

1. Type the following in a Controller:
```js
`com.mycompany.myapp.${someValue}`
```
2. Run the linter
2. WrongFilePathLinter error is triggered saying it cannot find a file under "com.mycompany.myapp.${someValue"

**Expected behavior**
I'd rather WrongFilePathLinter skip this error.

**Screenshots**
N/A

**Desktop (please complete the following information):**

- OS: macOS
- UI5 version: SAPUI5 1.118.0
- ui5plugin-linter version: 1.11.1 (latest in VSCodeUI5Plugin at the time of writing)

**Additional context**
This may also happen with normal string concatenation. E.g.:
```js
"com.mycompany.myapp." + someValue
```

**Project attachment**
N/A. I don't think it is needed.

**Comment by iljapostnovs** [comment]
Fixed, please check

**Comment by netux** [comment]
Solved. Thank you!
